ibcutils/testing/README.md

```diff
@@ -0,0 +1,326 @@
+# IBC Testing Package
+
+## Components
```

Comment:
IIRC, you said that we didn't end up using this and went for rust testing suite validation instead. should we remove this testing suite if it's not running?

Reply:
As I mentioned, there are some valuable tests using this package, this directory should now only contain alive code.